### ISSUE TITLE
Braintree - Authorization and Capture

### ISSUE THREAD

**Issue by angelleye** [angelleye/paypal-woocommerce#607]
Looks like we never got our Auth and Cap functionality built into the Braintree gateway.  Need to go ahead and do that.

https://www.angelleye.com/support/scp/tickets.php?id=809

**Comment by kcppdevelopers** [comment]
@angelleye Braintree does not support Authorization and Capture like PayPal. please read more information [here](https://articles.braintreepayments.com/control-panel/transactions/managing-authorizations)

**Comment by angelleye** [comment]
I'm a little confused.  Based on what I'm reading it sounds like they do support Auth and Capture.  The doc you linked to is just discussing how to handle expiration and verifying whether funds will be available.

Or, is it that this treats it more like a reference transaction / card vault...similar to the built in WC tokenized payments we just finished..??

If that's the case, we could go ahead and do that for Braintree so that we can still get auth / capture functionality working.

For example, if they have the gateway settings set to Auth, the order could complete and simply setup the card vault with Braintree.

Then when they "capture" it would use the vaulted card details to process the payment.  

So it's not a true auth/capture the way we did elsewhere, but the experience could still be the same overall.

Does that make sense?
